### DISCUSSION TITLE
Bulk Link creation

### DISCUSSION THREAD

**Discussion by rarchitgupta** [CSC510SEFall24/URL-Shortener-2.0#14] [General]
Being able to create bulk links was a suggested feature via the poster

Can be implemented as follows:
- Convert .txt files to JSON data (or directly uploading a JSON file)
- Iterating through all URL's and pass to already existing link shorten functions
- Has to be tested with files

**Comment by rarchitgupta** [comment]
Assigning @vyompatel77 to the same

**Comment by vyompatel77** [comment]
Implemented the requirements defined in the first two points in the code. 
Test cases are showing a few minor errors working on it.
